.github/workflows/build.yml

```diff
@@ -95,8 +95,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        #runs on the lowest and highest supported versions (change this to 8.4 once psalm supports it)
-        php: [8.1, 8.3]
+        php: [8.1, 8.4]
```

Comment:
We can now run in PHP 8.4 as we are not using Psalm which does not support this version